### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by OAyomide** [akkoro/assemblylift#1]
Can there be an outline of what to get started with?

It'd be nice to have contribution guide/to-do list even if its a simple list

**Comment by dotxlem** [comment]
@OAyomide For sure! I meant to write something up yesterday, but I got a little sidetracked. I'll try and get that up at some point today!

**Comment by dotxlem** [comment]
@OAyomide I've updated the README -- I have a few things listed that I think should be in place prior to a 0.1 release.

I'll start filing issues here throughout the day with some more details on each :)

**Comment by OAyomide** [comment]
Swell!! good one.
